service/server_grpc.go

```diff
@@ -3,8 +3,8 @@ package service
 import (
 	"context"
 
-	"github.com/go-kit/kit/log"
 	grpctransport "github.com/go-kit/kit/transport/grpc"
+	"github.com/kolide/launcher/log"
```

Comment:
This is a breaking change for servers. since the `log.Logger` in this file has changed to a much larger interface, any `NewGRPCServer` will require to instantiate a new launcher/log.Logger rather than the go-kit one. I'm not sure that's a good change.